app/views/bankdetails/bank_account_types.scala.html

```diff
@@ -55,13 +55,6 @@
                     labelText = "bankdetails.accounttype.lbl.03",
                     value = "03"
                 )
-                @if(count <= 1) {
-                    @radio(
-                        f = f("bankAccountType"),
-                        labelText = "bankdetails.accounttype.lbl.04",
-                        value = "04"
-                    )
-                }
             }
```

Comment:
There are additional tests in here which relate to if a person already has bank accounts which need to be removed.